Bottom: 988255f2f3c6c047c77cbc10263dbd4e4c569194
Top:    1cd85c568bd3397bcfb0ff8fd7395658af91c95e
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-08 15:26:45 +0800

Refresh of autofs-5.0.7-fix-submount-tree-not-all-expiring.patch

---

diff --git a/lib/master.c b/lib/master.c
index b53f7fd..173066e 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -913,10 +913,11 @@ int master_notify_submount(struct autofs_point *ap, const char *path, enum state
 			char *this_path = strdup(this->path);
 			if (this_path) {
 				mounts_mutex_unlock(ap);
-				if (!master_notify_submount(this, path, state)) {
+				/*if (!master_notify_submount(this, path, state)) {
 					free(this_path);
 					return 0;
-				}
+				}*/
+				master_notify_submount(this, path, state);
 				mounts_mutex_lock(ap);
 				if (!__master_find_submount(ap, this_path)) {
 					free(this_path);
